ESP8266Interface.cpp

```diff
@@ -26,7 +26,7 @@
 #define ESP8266_SEND_TIMEOUT    500
 #endif
 #ifndef ESP8266_RECV_TIMEOUT
-#define ESP8266_RECV_TIMEOUT    0
+#define ESP8266_RECV_TIMEOUT    500
```

Comment:
Some quick math, 5 symbols (+IPD:) / 115200 baud = 0.043 ms.

500ms > 0.043ms so this looks good 👍